R/postprocess.R

```diff
@@ -324,7 +324,8 @@ enw_pp_summary <- function(fit, diff_obs,
 #' posterior <- enw_posterior(fit$fit[[1]], var = "expr_lelatent_int[1,1]")
 #' enw_quantiles_to_long(posterior)
 enw_quantiles_to_long <- function(posterior) {
-  long <- melt(as.data.table(posterior),
+  posterior <- coerce_dt(posterior)
```

Comment:
feels like we should take advantage of the coerce_dt arguments to ensure the incoming object has the relevant features.

but i'm not sure what those features are, aside from quantile markers (and coerce_dt doesn't support pattern...yet) - @seabbs ideas?

Reply:
no no ideas but I think we should move discussion of this to a new issue/PR pair as it feels out of context for this one.

Reply:
If going to move to a new issue/PR, I'll go ahead an resolve this